File: backend/dotnet-webapi/src/WorldCup.Api/Dtos/ItineraryStopDto.cs
using WorldCup.Api.Models;

namespace WorldCup.Api.Dtos;

public class ItineraryStopDto
{
    public int StopNumber { get; set; }
    public City City { get; set; } = null!;
    public MatchWithCityDto Match { get; set; } = null!;
    public double DistanceFromPrevious { get; set; }
}

File: backend/java-spring/src/main/java/com/unosquare/worldcup/dto/ItineraryStopDTO.java
package com.unosquare.worldcup.dto;

import com.unosquare.worldcup.model.City;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ItineraryStopDTO {
    private int stopNumber;
    private City city;
    private MatchWithCityDTO match;
    private double distanceFromPrevious;
}
